testsV2/ut_repyv2api_filecloseconcurrency.r2py

```diff
@@ -1,9 +1,12 @@
 """
 This unit test checks the behavior of the fileobject when
-close is called concurrently with a read/write.
+`close` is called concurrently with `readat`/`writeat`.
```

Comment:
Should we add a comment here about the behavior we are checking, or is this common sense? E.g. something like:

```
# A `writeat`/`readat` operations will be finished, even if the filehandle is closed in a concurrent thread
```